src/components/pages/PowerheadPage/test.js

```diff
@@ -0,0 +1,235 @@
+		.map(el => el.children[2].children[0].text);
+
+	return { reportCount, internals, externals, bench, booked, confirmed_revenue, overruns };
+}
+
+describe('augmentReportsWithMetadata', () => {
+
+	it('should be a function', () => {
+		expect(augmentReportsWithMetadata).to.be.a('function');
+	});
+
+	it('should add costs to reports', () => {
+		const augmentedReports = augmentReportsWithMetadata(mockData.results);
+		expect(augmentedReports[0].costs).to.eql([55000, 55000, 55000, 55000]);
+		expect(augmentedReports[1].costs).to.eql([88000, 88000, 88000, 88000]);
+	});
+
+	it('should add financialsSum to reports', () => {
+		const augmentedReports = augmentReportsWithMetadata(mockData.results);
+		expect(augmentedReports[0].financialsSum).to.eql([ 1005, 1005, 1005, 1005 ]);
+		expect(augmentedReports[1].financialsSum).to.eql([ 2010, 2010, 2010, 2010, ]);
+	});
+	
+	it('should add maxFinancialsVal to reports', () => {
+		const augmentedReports = augmentReportsWithMetadata(mockData.results);
+		expect(augmentedReports[0].maxFinancialsVal).to.eql(55000);
+		expect(augmentedReports[1].maxFinancialsVal).to.eql(88000);
+	});
+
+	it('should add companyWideMaxFinancialsVal to reports', () => {
+		const augmentedReports = augmentReportsWithMetadata(mockData.results);
+		expect(augmentedReports[0].companyWideMaxFinancialsVal).to.eql(88000);
+		expect(augmentedReports[1].companyWideMaxFinancialsVal).to.eql(88000);
+	});
+});
+
+describe('combineReports', () => {
+
+	it('should be a function', () => {
+		expect(combineReports).to.be.a('function');
+	});
+
+	it('should combine array of n reports to array of 1 report', () => {
+		const reports = mockData.results;
+		const combinedReportArray = combineReports(reports);
+		expect(combinedReportArray).to.have.length.of(1);
+	});
+
+	describe('combined report', () => {
+		const combinedArray = combineReports(mockData.results);
+		const combined = combinedArray[0];
+		
+		it('should have costs calculated correctly', () => {
+			expect(combined.costs).to.eql([143000, 143000, 143000, 143000]);
+		});
+		it('should have financialsSum calculated correctly', () => {
+			expect(combined.financialsSum).to.eql([3015, 3015, 3015, 3015]);
+		});
+		it('should have maxFinancialsVal calculated correctly', () => {
+			expect(combined.maxFinancialsVal).to.eql(143000);
+		});
+		it('should have companyWideMaxFinancialsVal calculated correctly', () => {
+			expect(combined.companyWideMaxFinancialsVal).to.eql(143000);
+		});
+	})
+});
+
+describe('PowerheadPageHTTP', () => {
+
+  it('should be a function', () => {
+	expect(PowerheadPageHTTP).to.be.a('function');
+  });
+
+  it('calling the HTTP should return a request with url', (done) => {
+	const props$ = Rx.Observable.just({});
+	const HTTPSource = Rx.Observable.empty();
+	const {request$, response$} = PowerheadPageHTTP({props$, HTTP: HTTPSource});
+	
+	request$.subscribe(req => {
+
+		expect( _.startsWith(req.url, `${API_PATH}/powerhead/`) ).to.be.true;
+		done();
+	});
+  });
```

Comment:
We have .eslintignore file that matches all of our test files. Maybe we should make a branch where we would remove the ignore and fix style errors in tests. We can always override certain rules for tests.